pkgs/development/python-modules/hyprpy/default.nix

```
@@ -0,0 +1,42 @@
+{
+  buildPythonPackage,
+  fetchFromGitHub,
+  lib,
+  pydantic,
+  pythonOlder,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "hyprpy";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "ulinja";
+    repo = "hyprpy";
+    tag = "v${version}";
+    hash = "sha256-b312PmJoVPT5Dt695JdTgDCVlm2LcD0hMmsRUqs3VcQ=";
+  };
+
+  pyproject = true;
+  disabled = pythonOlder "3.7";
+  build-system = [
+    setuptools
+  ];
+
+  dependencies = [
+    pydantic
+  ];
+
+  doCheck = false;
+  pythonImportsCheck = [ "hyprpy" ];
+
+  meta = with lib; {
```

```suggestion
  meta = {
```
with lib;  is outdate